src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveClimateControlScheduleCommandClass.java

```diff
@@ -61,10 +72,80 @@ public void handleApplicationCommandRequest(SerialMessage serialMessage, int off
                 this.getVersion());
         int command = serialMessage.getMessagePayloadByte(offset);
         switch (command) {
+            case SCHEDULE_CHANGED_GET:
+                logger.debug("Answering with noop SCHEDULE_CHANGED_REPORT");
+                getController().enqueue(getScheduleChangedReportMessage(SCHEDULE_CHANGE_TEMPORARILY_DISABLED));
+                break;
+            case SCHEDULE_OVERRIDE_REPORT:
+                OverrideType overrideType = getOverrideTypeFor(serialMessage.getMessagePayloadByte(offset + 1) & 0x03);
+                ScheduleState scheduleState = getScheduleStateFor((byte) serialMessage.getMessagePayloadByte(offset + 2));
+                logger.info("NODE {} reported: Override type: {}, ScheduleState: {}", this.getNode().getNodeId(), overrideType, scheduleState);
+                break;
             default:
                 logger.warn(String.format("NODE %d: Unsupported Command %d for command class %s (0x%02X).",
                         this.getNode().getNodeId(), command, this.getCommandClass().getLabel(),
                         this.getCommandClass().getKey()));
         }
     }
+
+    public ScheduleState getScheduleStateFor(byte messagePayloadByte) {
+        if (messagePayloadByte == 0x79) return new ScheduleState(ScheduleStateState.FROST_PROTECTION,0);
```

Comment:
Please ensure that the formatter is run on the code - this doesn't look like it's formatted in the standard way.